components/mailgun/sources/new-bounce/new-bounce.js

```diff
@@ -0,0 +1,50 @@
+const common = require("../common-webhook");
+const { mailgun } = common.props;
```

Comment:
Is this used? It looks like you destructure `...common.props` within `props` already and I don't see a reference to it elsewhere.